lib/arel/visitors/spanner.rb

```diff
@@ -14,15 +14,20 @@ def compile node, collector = Arel::Collectors::SQLString.new
 
       private
 
+      BIND_BLOCK = proc { |i| "@p#{i}" }
```

Comment:
This generates named parameters in the format `@p1, @p2, ...`. This replaces the previously named parameters in the form `@id_1, @name_2, ...`, because there are some queries that are generated by ActiveRecord without a name for the parameter available.